roles/confluent.kafka_connect_replicator/templates/kafka-connect-replicator.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description= Kafka Connect Replicator
+Documentation=http://docs.confluent.io/
+After=network.target confluent-server.target
+
+[Service]
+Type=simple
+User=cp-kafka-connect-replicator
+Group=confluent
+ExecStart=/bin/replicator --consumer.config /etc/kafka-connect-replicator/kafka-connect-replicator-consumer.properties --producer.config /etc/kafka-connect-replicator/kafka-connect-replicator-producer.properties --cluster.id replicator --replication.config /etc/kafka-connect-replicator/kafka-connect-replicator.properties --consumer.monitoring.config=/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties --producer.monitoring.config=/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties
```

Comment:
make use of variables for these paths:
/etc/kafka-connect-replicator/kafka-connect-replicator-consumer.properties
/etc/kafka-connect-replicator/kafka-connect-replicator-producer.properties
etc

Reply:
We shouldn't be using variables here, this is the systemD unit file which should not be modified or edited.  If you want to make changes, they would go into the overrides file.  The only reason why we have access to this file is because I had to write it as Replicator does not ship with one.

Reply:
You are correct, we do not want to use overrideable variables here, but the vars defined in confluent.variables/vars/main.yml are not customizable. This comment is just to avoid hardcoding things. Here's the snippet:

```
kafka_connect_replicator:
  replication_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator.properties"
  consumer_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator-consumer.properties"
  producer_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator-producer.properties"
  interceptors_config_file: "{{ a
```

and further if you hardcode the full path to /etc/ then it wont work with the tar install which puts everything in /opt/confluent/

```suggestion
ExecStart=/bin/replicator --consumer.config {{kafka_connect_replicator.consumer_config_file}} --producer.config {{kafka_connect_replicator.producer_config_file}} --cluster.id replicator --replication.config {{kafka_connect_replicator.replication_config_file}} --consumer.monitoring.config=/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties --producer.monitoring.config=/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties
```

also I am noticing the same file `/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties` is being passed to `--consumer.monitoring.config` and `--producer.monitoring.config` is that intentional?

Reply:
is the cluster.id something that should be customizable?

Reply:
@domenicbove Understood about using the variables with archive install, that makes sense.  

The same file is used for monitoring, that is intentional as you can put both configs in the same file but still need to specify separate properties.  You could break them out as separate files, but they are identical and thus redundant.

cluster.id should be configurable, I have updated this.